src/pages/trader-tools/_trading-tools.tsx

```diff
@@ -1,13 +1,28 @@
 import React from 'react'
 import styled from 'styled-components'
-import PropTypes from 'prop-types'
 import { graphql, useStaticQuery } from 'gatsby'
 import { isIndexEven } from 'common/utility'
 import { Container, SectionContainer, Flex, Show } from 'components/containers'
 import { Header, Text, QueryImage } from 'components/elements'
 import { LinkButton } from 'components/form'
 import device from 'themes/device'
 
+type TradingToolsProps = {
+    reverse?: boolean
+    tools: {
+        title: JSX.Element
+        subtitle: JSX.Element
+        link: { text: JSX.Element; route: string }
+        image_name: string
+        image_alt: string
+    }[]
```

Comment:
Create a tools type instead then apply it here 

```
 tools: ToolsType[]
 ```

Reply:
Done